src/app/__test__/tasks_drag.test.js

```diff
@@ -0,0 +1,133 @@
+    <ul class="container-list"></ul>
+  </div>`;
+
+  const ul = document.querySelector('.container-list');
+
+  const listTask = [
+    'Make Candy',
+    'Buy Ticket',
+    'Send Video',
+  ];
+
+  listTask.forEach((element) => add(element, ul));
+
+  test('the task should change his value in the storage', () => {
+    // The first task is Make Candy now lets change that
+    const newDescription = 'Change Task description';
+    const descriptionUpdate = updateItems(0, newDescription, ul);
+
+    expect(descriptionUpdate[0].description).toBe('Change Task description');
+  });
+
+  test('The task should change in the ul', () => {
+    const firstElementofTheList = ul.children[0];
+
+    expect(firstElementofTheList.textContent)
+      .toContain('Change Task description');
+  });
+});
+
+describe('Test for the update completed method', () => {
+  const ul = document.querySelector('.container-list');
+
+  test('completed property should change to TRUE', () => {
+    const indexOfElementCompleted = 1;
+
+    const updateCompleted = checkCompleted(ul, indexOfElementCompleted);
+
+    expect(updateCompleted[indexOfElementCompleted].completed).toBe(true);
+  });
+});
+
+describe('Test Drag/Drop functionality', () => {
+  const ul = document.querySelector('.container-list');
+
+  test('change the index of the elements on drag', () => {
+    const dragElement = 0;
+    const dropElement = 2;
+
+    const newStorage = drag.swap(dragElement, dropElement, ul);
+
+    expect(newStorage).toEqual(
+      expect.arrayContaining([
+        {
+          description: 'Send Video',
+          completed: false,
+          index: 0,
+        },
+        {
+          description: 'Buy Ticket',
+          completed: true,
+          index: 1,
+        },
+        {
+          description: 'Change Task description',
+          completed: true,
+          index: 2,
+        },
+      ]),
+    );
+  });
+
+  test('change the index of the elements on drag again', () => {
+    const dragElement = 1;
+    const dropElement = 2;
+
+    const newStorage = drag.swap(dragElement, dropElement, ul);
+
+    expect(newStorage).toEqual(
+      expect.arrayContaining([
+        {
+          description: 'Send Video',
+          completed: false,
+          index: 0,
+        },
+        {
+          description: 'Change Task description',
+          completed: true,
+          index: 1,
+        },
+        {
+          description: 'Buy Ticket',
+          completed: true,
+          index: 2,
+        },
+      ]),
+    );
+  });
+});
+
+describe('Test remove completed task', () => {
+  const ul = document.querySelector('.container-list');
+  test('remove all completed elements', () => {
+    const storageWithoutCompleted = remove.completed(ul);
+    expect(storageWithoutCompleted).toEqual(
+      expect.arrayContaining([
+        {
+          description: 'Send Video',
+          completed: false,
+          index: 0,
+        },
+      ]),
```

Comment:
- [ ] There should be one expect per test.

Reply:
https://jestjs.io/docs/expect#expectarraycontainingarray